ops/install-feature-service-db.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+readonly location="$1"
+readonly resource_group="$2"
+
+readonly pg_dump="https://fortiscentral.blob.core.windows.net/locations/feature-service.v1.sql.gz"
+readonly pg_admin="${FEATUREDB_ADMIN:-fortisadmin}"
+readonly pg_name="${FEATUREDB_NAME:-fortis-feature-service-db}"
+readonly pg_tier="${FEATUREDB_TIER:-Basic}"
+readonly pg_compute="${FEATUREDB_COMPUTEUNITS:-50}"
+readonly pg_version="${FEATUREDB_POSTGRESVERSION:-9.6}"
+readonly pg_password="$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c"${PASSWORD_COMPLEXITY:-32}")"
+
+if ! (command -v jq >/dev/null); then sudo apt-get install -y jq; fi
+if ! (command -v psql >/dev/null); then sudo apt-get install -y postgresql postgresql-contrib; fi
+
+echo "!!!! Postgres password for ${pg_admin} set to '${pg_password}' !!!!"
```

Comment:
I'd just generate a password guid in the deployment script which we'd propagate in the kubernetes `ConfigMap`